lib/block-editor-settings.php

```diff
@@ -74,9 +78,74 @@ function gutenberg_get_block_editor_settings( $settings ) {
 
 	$settings['styles'] = array_merge( $global_styles, get_block_editor_theme_styles() );
 
-	// Copied from get_block_editor_settings() at wordpress-develop/block-editor.php.
 	$settings['__experimentalFeatures'] = gutenberg_get_global_settings();
+	// These settings may need to be updated based on data coming from theme.json sources.
```

Comment:
There's no similar handler in the core; let's double-check if this is really needed.

Reply:
This is copied directly from https://github.com/WordPress/wordpress-develop/blob/23b007b126fa73211b7db56a38e78e601e4abbc6/src/wp-includes/block-editor.php#L443.